django/conf/global_settings.py

```diff
@@ -587,6 +587,32 @@
     # 'django.contrib.staticfiles.finders.DefaultStorageFinder',
 ]
 
+FILE_STORAGES = {
+    'media': {
```

Comment:
@aaugustin @timgraham media backend will have `media` key in settings, not `default`? Looks not quite consistent with the current settings.